lib/realtime/tenants.ex

```diff
@@ -310,7 +310,7 @@ defmodule Realtime.Tenants do
   @doc """
   Lists all active tenants from the ETS table named `:active_tenants`.
   """
-  @spec track_active_tenant(String.t()) :: list({String.t(), NaiveDateTime.t()})
+  @spec list_active_tenants() :: list({String.t(), NaiveDateTime.t()})
   def list_active_tenants() do
     :ets.tab2list(:active_tenants)
   end
```

Comment:
could it be an issue with having a large number of tenants?

Reply:
we're seeing 3k max across the cluster so per node, worse case scenario, we would have 3k entries at this point in time which doesn't seem that big for a tuple of `{String.t(), DateTime.t()}`

also we only run this command once every ~4 hours so it should not be a big impact but we can look into have some sorte of offset to batch it 🤔 

but would rather do it if required. Worst node we saw had 300 entries